.editorconfig

```diff
@@ -7,3 +7,6 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
+
+[package.json]
```

Comment:
I'm fine requiring this for all `json` files.

Reply:
Yeah I has just preventing my ide from making a mess